### PR TITLE
[XM] Add missing SafariServices binding

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

**Comment by chamons** [comment]
I checked every header by hand after misunderstanding the initial request, then double checked xtro.

File: src/safariservices.cs [hunk 1]
```diff
@@ -123,6 +123,10 @@ interface SFSafariApplication
 		[Static]
 		[Export ("setToolbarItemsNeedUpdate")]
 		void SetToolbarItemsNeedUpdate ();
+
+		[Static]
+		[Export ("showPreferencesForExtensionWithIdentifier:completionHandler:")]
+		void ShowPreferencesForExtensionWithIdentifier (string identifier, [NullAllowed] Action<NSError> completionHandler);
```
Comment (spouliot) [inline src/safariservices.cs]: remove `WithIdentifier` suffix - it's already implied by the parameters

**Comment by chamons** [comment]
I did not comment in the extension APIs since we don't have bandwidth currently to build the template to test.

File: src/safariservices.cs [hunk 2]
```diff
@@ -241,5 +245,23 @@ interface SFSafariExtensionViewController
...
 	}
+
+//	TODO - Needs Safari Extension support to test
+// 	[Mac (10,12)]
+// 	[BaseType (typeof(NSObject))]
+// 	interface SFSafariExtensionManager
+// 	{
+// 		[Static]
+// 		[Export ("getStateOfSafariExtensionWithIdentifier:completionHandler:")]
+// 		void GetStateOfSafariExtensionWithIdentifier (string identifier, Action<SFSafariExtensionState, NSError> completionHandler);
```
Comment (spouliot) [inline src/safariservices.cs]: same wrt `WithIdentifier`

File: src/safariservices.cs [hunk 3]
```diff
@@ -241,5 +245,23 @@ interface SFSafariExtensionViewController
 	interface SFSafariExtensionHandler : NSExtensionRequestHandling, SFSafariExtensionHandling
 	{
 	}
+
+//	TODO - Needs Safari Extension support to test
```
Comment (spouliot) [inline src/safariservices.cs]: add them to `osx.pending` so xtro does not report them in the unclassified list

Comment (chamons) [inline src/safariservices.cs]: Good call.

**Pull request merged**